specification/data-semantic-conventions.md

```diff
@@ -73,46 +101,48 @@ attribute names.
 
 | Attribute name | Notes and examples                                           | Required? |
 | :------------- | :----------------------------------------------------------- | --------- |
-| `component`    | Database driver name or database name (when known) `"JDBI"`, `"jdbc"`, `"odbc"`, `"postgreSQL"`. | Yes       |
+| `component` | Always the string `"db"` | Yes       |
 | `db.type`      | Database type. For any SQL database, `"sql"`. For others, the lower-case database category, e.g. `"cassandra"`, `"hbase"`, or `"redis"`. | Yes       |
-| `db.instance`  | Database instance name. E.g., In java, if the jdbc.url=`"jdbc:mysql://db.example.com:3306/customers"`, the instance name is `"customers"`. | Yes       |
+| `db.instance`  | Database instance name. E.g., In Java, if the jdbc.url=`"jdbc:mysql://db.example.com:3306/customers"`, the instance name is `"customers"`. | Yes     |
 | `db.statement` | A database statement for the given database type. Note, that the value may be sanitized to exclude sensitive information. E.g., for `db.type="sql"`, `"SELECT * FROM wuser_table"`; for `db.type="redis"`, `"SET mykey 'WuValue'"`. | Yes       |
+| `db.url` | The connection string used to connect to the database | Yes       |
+| `db.clientlib` | Database driver name or database client library name (when known), e.g., `"JDBI"`, `"jdbc"`, `"odbc"`, `"com.example.postresclient"`. | No       |
+| `db.tech` | Database technology, e.g. `"PostgreSQL"`, `"SQLite"`, `"SQL Server"` | No       |
+| `db.resultcount` | An integer specifying the number of results returned. | No       |
+| `db.roundtripcount` | An integer specifying the number of network roundtrips while executing the request. | No       |
 | `db.user`      | Username for accessing database. E.g., `"readonly_user"` or `"reporting_user"` | No        |
 
-For database client calls, peer information can be populated and interpreted as
-follows:
-
-| Attribute name  | Notes and examples                                           | Required |
-| :-------------- | :----------------------------------------------------------- | -------- |
-| `peer.address`  | JDBC substring like `"mysql://db.example.com:3306"`          | Yes      |
-| `peer.hostname` | Remote hostname. `"db.example.com"`                          | Yes      |
-| `peer.ipv4`     | Remote IPv4 address as a `.`-separated tuple. E.g., `"127.0.0.1"` | No       |
-| `peer.ipv6`     | Remote IPv6 address as a string of colon-separated 4-char hex tuples. E.g., `"2001:0db8:85a3:0000:0000:8a2e:0370:7334"` | No       |
-| `peer.port`     | Remote port. E.g., `80` (integer)                            | No       |
-| `peer.service`  | Remote service name. Can be database friendly name or `"db.instance"` | No       |
+Additionally the `peer.name` attribute from the [network attributes][] is required and `peer.ip` and `peer.port` are recommended.
 
-## gRPC
+## General RPC
```

Comment:
Maybe use Remote procedure call here instead General RPC. Or is there also some non general RPC?